server/config/db.config.js

```diff
@@ -0,0 +1,8 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('my_learn', 'root', 'experion@123', {
```

Comment:
Try to read these data from an env file.

Reply:
Issue resolved.